lib/content-info/index.js

```diff
@@ -9,17 +9,59 @@ var fileType = require('file-type'),
     /**
      * @private
      * @const
-     * @type {string}
+     * @type {String}
      */
     E = '',
 
     /**
      * @private
      * @const
-     * @type {string}
+     * @type {String}
+     */
+    DOT = '.',
+
+    /**
+     * @private
+     * @const
+     * @type {Number}
+     */
+    Number_16 = 16,
```

Comment:
`SIXTEEN`, for consistency with the rest of the codel

Reply:
@kunagpal for other numbers, we added figures in variable

Reply:
Also constants are all caps! WTX is happening with these little things, why isn't the pattern recognised????

Reply:
Also, we usually don't need constants for simple numbers that are used as arguments to base JS functions. Constants are defined when the mean something. E.g.

- For defining execution timeout, `EXEC_TIMEOUT=2345` instead of `NUMBER_2345=2345`.
- Things like parseInt, etc can simply have the literal passed on.